wherehows-frontend/app/controllers/api/v1/Jira.java

```diff
@@ -51,7 +51,7 @@ public static Result getJiraTickets(String managerId) {
     List<JiraTicket> tickets = JiraDAO.getUserTicketsByManagerId(managerId);
     if (tickets != null && tickets.size() > 0) {
       for (JiraTicket ticket : tickets) {
-        if (ticket.ticketComponent.equalsIgnoreCase(HEADLESS_COMPONENT)) {
+        if (ticket.getTicketComponent().equalsIgnoreCase(HEADLESS_COMPONENT)) {
```

Comment:
I am not sure if ticket.getTicketComponent can return a null. It will throw an exception in null case.
Better to do HEADLESS_COMPONENT.equalsIngnoreCase(ticket.getTicketConponent()) here?